modules/chapter1/pages/section5.adoc

```diff
@@ -0,0 +1,91 @@
+= Text Generation Inference Server (TGIS)
+
+The TGIS implementation in RHOAI is a fork of the HuggingFace TGIS repository. 
+
+More details around the reasons for the fork are available in the https://github.com/opendatahub-io/text-generation-inference[TGIS Repo].
+
+On RHOAI TGIS is used as
+
+* The serving runtime for models using the CAIKIT framework.
+* A standalone KServe runtime.
+
+We're only going to focus on the TGIS-KServe usecase here.
+
+== TGIS Overview
+TGIS is a model serving runtime written in Rust which serves _PyTorch_ models via a _gRPC_ interface. It only supports the _SafeTensors_ model format.
```

Comment:
For more info about safe tensors, we might want to add a reference to the huggingface docs https://huggingface.co/docs/safetensors/index